src/components/landing-layout/footer/language-picker.tsx

```diff
@@ -41,16 +41,16 @@ const languages = {
 }
 
 const findLanguageCode = (wantedLanguage: string): string => {
-  let foundLanguage = Object.keys(languages)
-                            .find((supportedLanguage) => wantedLanguage === supportedLanguage)
-  if (!foundLanguage) {
-    foundLanguage = Object.keys(languages)
-                          .find((supportedLanguage) => wantedLanguage.substr(0, 2) === supportedLanguage)
-  }
-  return foundLanguage || ''
+  return (
+    Object.keys(languages)
+          .find((supportedLanguage) => wantedLanguage === supportedLanguage) ??
+    Object.keys(languages)
+          .find((supportedLanguage) => wantedLanguage.substr(0, 2) === supportedLanguage) ??
+    ''
+  )
```

Comment:
I'd suggest adding some comments here. It's not the easiest to read…

Reply:
Added jsdoc